src/common/Endian.h

```diff
@@ -119,6 +119,12 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 # error The file boost/detail/endian.hpp needs to be set up for your CPU type.
 #endif
 
+// forward declaration
+namespace Util {
+	template<typename ToT, typename FromT>
+	ToT bit_cast(FromT from);
+}
+
```

Comment:
document

Reply:
This is just a forward declaration to make DaemonArchitecture compile. The documentation is in Util.h